### ISSUE TITLE
Team update - Jun 21, 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#219]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by steff456** [comment]
**Thanks I'd like to give 🙌**
- @gabalafou for doing the final review of the Github Light PR

**Updates from last week :heavy_check_mark:**
- Merged https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/49
- Waiting review for notebook authoring guidelines https://github.com/jupyter/accessibility/pull/127
- Waiting review for the ARIA and WCAG intro https://github.com/jupyter/accessibility/pull/126
- Opened PR for Github dark https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/53
- Started writing the docs for the 400% Zoom audit
- Mike update is that he was doing the PR in his free time, and now he doesn’t have the bandwidth to finish it soon. 
- For next week I'll bring a plan per week for the internship

**Challenges I faced and things I'd like assistance with 🙏**
- I'm still waiting for more feedback on the notebook authoring guide, so I didn't made a lot of progress in that front this week.

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- What is missing in https://github.com/jupyter/accessibility/pull/126 to be merged?
